canonical-websites/www.ubuntu.com
# Remove black and flake8 dependencies

We can't install black and flake8 during the build process. So we should remove them from `requirements.txt`.

I've also updated the `./run` script, and done some other tidying.

QA
--

``` bash
./run clean
./run build
./run test
./run
```

Check the site works.
